spec/features/projects/edit_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe 'Editing project' do
+  before do
+    @project = create :project,
+                      description: "# Here's some info about the project\n\nBla bla bla.",
+                      customer: "# Here's customer about the project\n\nBla bla bla."
+    login_as(create :admin)
+  end
+
+  it 'edit a project' do
+    visit edit_project_path(@project)
+
+    fill_in 'project_name',         with: ''
+    fill_in 'project_description',  with: 'The project description'
+    fill_in 'project_customer',     with: 'The project customer'
+
+    click_button 'Update Project'
+
+    expect(page).to have_content 'Alert: Project could not be updated.'
+
+    fill_in 'project_name',       with: 'The new Project'
+
+    click_button 'Update Project'
```

Comment:
Ab hier solltest du besser testen, dass die Änderungen in der DB angekommen sind! Wie bei den Pages:

```ruby
    expect {
      click_button 'Update Page'
      @page.reload
    } .to  change { @page.title }.to('A new title')
      .and change { @page.navigation_title }.to('A new navigation title')
      .and change { @page.parent }.from(old_page_parent).to(new_parent_page)
      .and change { @page.position }.from(1).to(2)
      .and change { @page.content }.to("A new content with a ![existing image](@image-some-existing-identifier) and a ![new image](@image-some-new-identifier)")
      .and change { @page.notes }.to('A new note')
      .and change { @page.images.count }.by(1)
      .and change { @page.images.first.file.file.identifier }.to('file.png')
      .and change { @page.images.first.identifier }.to('some-existing-identifier')
      .and change { @page.images.last.file.file.identifier }.to('file.png')
      .and change { @page.images.last.identifier }.to('some-new-identifier')
```

Reply:
@retoinniger: Bitte erledigen!